diagnostic.md

```diff
@@ -6,72 +6,75 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The allow data to persist once a client's browser is closed and to allow users to access data from different clients.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+The model.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+The controller.
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Because they are no longer needed for the most part within the newer ansynchronous model.
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+Create, retrieve, update, delete.
 ```
```

Comment:
Create, Read, Update, Destroy.  You have the concept behind each one, but it's important to know the actual word.  I assure you this WILL come up on job interviews and they are looking this exactly "Create, Read, Update, Destroy"